NEWS.md

```diff
@@ -37,6 +37,7 @@ New library functions
 
 * `logrange(start, stop; length)` makes a range of constant ratio, instead of constant step ([#39071])
 * The new `isfull(c::Channel)` function can be used to check if `put!(c, some_value)` will block. ([#53159])
+* `waitany(tasks; throw=false)` and `waitall(tasks; failfast=false, throw=false)` which wait multiple tasks at once ([#53341])
```

Comment:
@vtjnash Is this the right place for these functions?

Reply:
Yes, this looks like a reasonable spot for it. Style nit: they usually end in a `.`, though someone goes through and does a formatting cleanup anyways during the release, so it isn't vital to get these exactly right

Reply:
I see. I'll fix this.